control/control.yaml

```diff
@@ -8,15 +8,52 @@ info:
 
 components:
   schemas:
+    Control.State:
+      description: >-
+        A container for the different types of states.
+      type: object
+      required: ['choice']
+      properties:
+        choice:
+          type: string
+          enum: [config_state, port_link_state, port_capture_state, flow_transmit_state]
+        config_state:
+          $ref: '#/components/schemas/Control.Config'
+        port_link_state:
+          $ref: '#/components/schemas/Control.PortLink'
+        port_capture_state:
+          $ref: '#/components/schemas/Control.PortCapture'
+        flow_transmit_state:
+          $ref: '#/components/schemas/Control.FlowTransmit'
```

Comment:
Can we rename,
- flow_transmit_state -> flow_state
- ControlFlowTransmit -> FlowState

Reasons:
- `Result` refers to it as FlowState
- Technically, the overall effect is not limited to transmit state